CMakeLists.txt

```diff
@@ -28,6 +28,11 @@ if(Kokkos_ENABLE_HIP AND NOT ARBORX_ENABLE_ROCTHRUST)
   message(WARNING "rocThrust is NOT enabled.\nThis will negatively impact performance on AMD GPUs.")
 endif()
 
+if(Kokkos_ENABLE_SYCL)
+  find_package(oneDPL REQUIRED)
```

Comment:
Is it installed and in your path on Iris?  If not we should have an option to enable/disable

Reply:
What are the cases when it's not faster?